src/App.js

```diff
@@ -1,29 +1,16 @@
-import logo from './logo.svg';
 import './App.css';
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that your remove this import of `App.css` because it could raise errors as this file is no longer in your repository.